bHY6yh-d1e515/cod-pu6y7d_bHY6yh-d1e515.xml

```diff
@@ -65,98 +65,98 @@
           domi<lb ed="#V" n="16" break="no"/>nus deus fortis et zelotes. </quote>
         </p>
         <p xml:id="bHY6yh-d1e515-d1e131"> /
-          <quote xml:id="bHY6yh-d1e515-Qd1e133"> Psal
-            <lb ed="#V" n="17"/>mista psalmus. 7. Dominus iudex iustus: for
+           ><ref xml:id="bHY6yh-d1e515-Qd1e133"> Psal
+          <lb ed="#V" n="17"/>mista psalmus. 7. </ref> <quote Dominus iudex iustus: for
```

Comment:
The **entire** <quote> element should be moved to where the quote beings. This means its "attribute" should be moved with it. The "attribute" in this case is the xml:id="bHY6yh-d1e515-Qd1e133"

You also seemed to have left the closing angle bracket in front of the `<ref>` giving us: 
```
><ref
```
Here's what the entry should look like at the end.
```
<p xml:id="bHY6yh-d1e515-d1e131"> /
    <ref>Psal
    <lb ed="#V" n="17" break="no"/>mista psalmus. 7. </ref> <quote xml:id="bHY6yh-d1e515-Qd1e133">Dominus 
    iudex iustus: for
    <lb ed="#V" n="18" break="no"/>tis et patiens. </quote>
</p>
```
Finally notice that "Psalmista" is broken by the line break. So here we needed to add the attribute `break="no"` as I have done above.